src/Shopway.Persistence/Registration/OpenTelemetryRegistration.cs

```diff
@@ -35,6 +35,9 @@ internal static IServiceCollection RegisterOpenTelemetry(this IServiceCollection
                 options.AddOtlpExporter(options => ConfigureOtlpCollectorExporter(options, openTelemetryOptions.OtlpCollectorHost));
             }
 
+            options.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(
+                serviceName: nameof(Shopway)));
```

Comment:
This is indeed an important part that I missed. Good job in finding it. 

Use the name and version from the openTelemetryOptions:
serviceName: openTelemetryOptions.ApplicationName, serviceVersion: openTelemetryOptions.Version